h/migrations/versions/8ae9d103551f_backfill_group_access_flags.py

```diff
@@ -0,0 +1,48 @@
+"""
+Backfill group access flags
+
+Revision ID: 8ae9d103551f
+Revises: af88524994ce
+Create Date: 2016-11-29 12:51:31.247598
+"""
+
+from __future__ import unicode_literals
+
+import enum
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '8ae9d103551f'
+down_revision = 'af88524994ce'
+
+
+class JoinableBy(enum.Enum):
+    authority = 'authority'
+joinable_type = sa.Enum(JoinableBy, name='group_joinable_by')
```

Comment:
Ok, so it's nice and clear that this migration script is only setting groups to joinable by 'authority', since that's the only value in its version of the enum. And it needs to have its own copy of the enum because we don't know what the real `h.models.group.JoinableBy` will look like (or if it'll even exist) in the future versions of the code that may be deployed when this migration script is run